lib/jobs.js

```diff
@@ -104,20 +162,20 @@ function stop(buildConfig) {
         })
         // If this is a waiting job
         .then(() => redis.lrem(`${waitingJobsPrefix}${jobId}`, 0, buildId))
-        .then(() => (started ? executor.stop(stopConfig) : null));
+        .then(() => (started ? schedule('stop', stopConfig) : null));
 }
 
 module.exports = {
     start: {
-        plugins: ['Retry', BlockedBy],
+        plugins: [Filter, 'Retry', BlockedBy],
```

Comment:
does `Filter` need to be in quotes?

Reply:
Filber is an object we required here: https://github.com/screwdriver-cd/queue-worker/pull/106/files/d65d77bad2447da150a9dbe88ad75292d295f83b#diff-6250865cbe8215106353c06e8a4f327dR9

Retry is in single quote is because it's built in plugin in node-resque, we just need to specify the name